### ISSUE TITLE
Upĝrade izpack from 5.0.6 to 5.0.7

### ISSUE THREAD

**Issue by adriens** [adriens/schemacrawler-izpack#1]

**Comment by adriens** [comment]
For now, this upgrade causes the build to fail as it seems that the `parent` attribute ine the `pack` element is not supported any more, see build stacktrace :

```
[INFO] --- izpack-maven-plugin:5.0.7:izpack (default) @ schemacrawler-lzpack ---
Apr 14, 2016 11:12:34 AM com.izforge.izpack.core.container.PlatformProvider provide
INFO: Detected platform: ubuntu_linux,version=3.19.0-32-generic,arch=x64,symbolicName=null,javaVersion=1.8.0_31
[INFO] Logging initialized at level 'INFO'
[WARNING] Maven property izpack.version could not be overridden
ERROR:  'cvc-complex-type.3.2.2: Attribute 'parent' is not allowed to appear in element 'pack'.'
ERROR:  'com.sun.org.apache.xml.internal.utils.WrappedRuntimeException: cvc-complex-type.3.2.2: Attribute 'parent' is not allowed to appear in element 'pack'.'
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 1.894 s
[INFO] Finished at: 2016-04-14T11:12:34+11:00
[INFO] Final Memory: 19M/270M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.codehaus.izpack:izpack-maven-plugin:5.0.7:izpack (default) on project schemacrawler-lzpack: Failure: Error in /home/salad74/NetBeansProjects/schemacrawler-izpack/src/izpack/install.xml at line 103, column 65 : javax.xml.transform.TransformerException: com.sun.org.apache.xml.internal.utils.WrappedRuntimeException: cvc-complex-type.3.2.2: Attribute 'parent' is not allowed to appear in element 'pack'. -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
```

**Comment by adriens** [comment]
Creating an issue on izpack jira instance.

**Comment by adriens** [comment]
Took a look at https://izpack.atlassian.net/wiki/display/IZPACK/Upgrading+Existing+Installers+from+IzPack+4.x+to+5.0
but found nothing

**Comment by adriens** [comment]
Issue created see https://izpack.atlassian.net/browse/IZPACK-1390

**Comment by adriens** [comment]
izpack Patch merged soon : https://github.com/izpack/izpack/pull/499
Will be fixed on izpack 5.0.8

**Comment by adriens** [comment]
5.0.8 is out since today, but it generates a regression, see https://izpack.atlassian.net/browse/IZPACK-1392
